internal/k8s/controller.go

```diff
@@ -724,6 +727,9 @@ func (c *AviController) SetupEventHandlers(k8sinfo K8sinformers) {
 			namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(svc))
 			if isSvcLb && !lib.GetLayer7Only() {
 				key = utils.L4LBService + "/" + utils.ObjKey(svc)
+				if !lib.ValidateSvcforClass(key, svc) {
```

Comment:
Is this validation required for delete event ?
Will this check not skip deletion of existing svcs w/o lb class

Reply:
Hi Priya, I have tested this scenario (What happens to invalid services when AKO is upgraded to use LBClass validation?)
All VSes associated with invalid LB services are automatically deleted and all VSes associated with valid LB services are kept. Hence this shouldnt be an issue

Reply:
Could you please check what happens to K8s Service object in this scenario ?